.github/workflows/coverage.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
           java-version: openjdk@1.11.0-2
 
       - name: Set up docker environment
-        run: docker-compose -f docker-compose.dev.yaml up -d
```

Comment:
I understand the idea of having multiple `docker-compose` in the same repository but maintain a single `docker-compose.yaml` will be easier. 

For deployments we will store everything including the Docker and K8S deployments of our project at https://github.com/hawks-atlanta/infrastructure

This is to prevent modifications of the pipelines every time.

Just make sure that after renaming it again to `docker-compose.yaml` you update:

- README
- Filename
- Test and coverage pipe

Reply:
This last change and we are ready to merge

Reply:
So, there is no problem with having the `pgadmin` container running in the test and coverage pipelines, right? I mean, the dashboard is useful during development but we're pulling an unused docker image in the pipelines.

Reply:
Don't worry about it